### ISSUE TITLE
BigSur 11.6 changed osx_vers relationship to kernel_vers

### ISSUE THREAD

**Issue by nieder** [fink/fink#236]
Usually, when Apple increments the minor release in BigSur they also incremented the darwin minor release
BigSur 11.0 = darwin20.1
BigSur 11.1 = darwin20.2

This pattern kept through BigSur 11.5 (darwin20.6). However, BigSur 11.6 is also darwin20.6 and this breaks anything that uses `Fink::Services::get_darwin_equiv`

https://github.com/TheSin-/fink/blob/f8dce532d570585b27bb8f496fd54f6bcc06e7f5/perlmod/Fink/Services.pm#L1439-L1456
```perl
sub get_darwin_equiv {
	my %darwin_osx = (
		'1' => '10.0',
	);
	my $kernel_vers = get_kernel_vers();
	my $kernel_vers_minor = get_kernel_vers_minor();
	if ($kernel_vers <= 19) {
		# darwin19 == 10.15
		return $darwin_osx{$kernel_vers} || '10.' . ($kernel_vers-4);
	} elsif ($kernel_vers == 20) {
		# darwin20.1 == 11.0
		# darwin20.2 == 11.1
		return $darwin_osx{$kernel_vers} || '11.' . ($kernel_vers_minor-1);
	} elsif ($kernel_vers >= 21) {
		# darwin21.0 == 12.0
		return $darwin_osx{$kernel_vers} || '12.' . ($kernel_vers_minor);
	}
}
```
How can we split 11.5 and 11.6? The obvious method is to use `sw_vers`, but then we'll eventually compare the output of sw_vers here with sw_vers in `get_osx_vers()`, so that seems silly.

@TheSin- @dmacks

**Comment by TheSin-** [comment]
what about a sub if in the BigSur block that checks kernel_vers_minor >= 6

```
} elsif ($kernel_vers == 20) {
		if ($kernel_vers_minor >= 6) {
			# darwin20.6 >= 11.6
			return $darwin_osx{$kernel_vers} || '11.' . $kernel_vers_minor;
		}
		# darwin20.1 == 11.0
		# darwin20.2 == 11.1
		return $darwin_osx{$kernel_vers} || '11.' . ($kernel_vers_minor-1);
```

or something like this?

**Comment by TheSin-** [comment]
that would also work.  So odd they would break it like that, our luck 11.7 will go to 20.8 again :\. Your version will handle that, so I'd go with that.

**Comment by nieder** [comment]
Fix with the merge of #66
